apps/mudita-center-e2e/src/page-objects/overview.page.ts

```diff
@@ -3,10 +3,13 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 export class OverviewPage extends Page {
+  public get overviewMenuItem() {
```

Comment:
All elements related to left navigation menu are common for all views and may be moved to separate object

Reply:
menu.page added :)